### ISSUE TITLE
Could you please share your result?

### ISSUE THREAD

**Issue by yc930401** [parksunwoo/show_attend_and_tell_pytorch#2]
Hi sunwoo,

I have trained a similar model with Coco Dataset. The loss went from around 27 to 16, but no matter how I change the learning rate (I tried lr from 0.001 to 0.0000001) after that, the loss remained stable, around 16. However, the natural language output are still very bad. Like this:

Ground Truth:  a baseball player for the chicago cubs stands at home plate .
Output:  such docks weathered weathered weathered brussell uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform uniform

Could you please share your result and the time you spent training it? Thanks a lot!
